cni/pkg/nodeagent/podcgroupns.go

```diff
@@ -373,3 +404,25 @@ func getPodUIDAndContainerIDFromCGroups(cgroups []Cgroup) (types.UID, string, er
 
 	return podUID, containerID, nil
 }
+
+func isHostNetns(proc fs.FS, foundIno, foundDev uint64) (bool, error) {
+	hf, err := fs.Stat(proc, path.Join("1", "ns", "net"))
+	if err != nil {
+		fmt.Printf("BOOP 1")
+		return false, err
+	}
+
+	hInode, hDev, err := GetInodeDev(hf)
```

Comment:
should we cache this in PodNetnsProcFinder? it shouldn't change, right?

Reply:
I had it in there at one point yeah, and then removed it.

Reasons I removed it/didn't cache it

1. We get both values from the same `Stat` result anyway, which we already have to do every-time to get the inode.
2. 999999.9% it won't change, but in theory it could.

There's no real benefit to caching it in the end tho since it saves 0 calls given how FileInfo works.

Reply:
Maybe I am misunderstanding. my impression is this is `for each proc { stat /proc/1/ns/net }`. Which seems like we could do

```
stat /proc/1/ns/net 
for each proc { .... }
```

at the very least?

Reply:
Oh, my bad - you mean the _host_ inode/dev - my bad. Got confused with something else I had in my head.

Yep.